cmd/query.go

```diff
@@ -135,7 +135,7 @@ func runQueryCmd(cmd *cobra.Command, args []string) {
 	}
 	defer initData.Cleanup(ctx)
 
-	var failures int
+	var failures, rowErrors int
```

Comment:
is there a need to distinguish between `failures` and `rowErrors` - can't we just use a single var?